### ISSUE TITLE
64 bit NVDA is bundled with 32 bit vccorlib140.dll

### ISSUE THREAD

**Issue by SaschaCowley** [nvaccess/nvda#18827]
### Steps to reproduce:

1. Download an x86_64 build of NVDA [such as this try build](https://github.com/nvaccess/nvda/actions/runs/17255353874/artifacts/3859542283).
2. Attempt to install it on a clean install of Windows.

### Actual behavior:

The launcher is unpacked, but running the temporary copy of NVDA fails with error [0xc000007b](https://www.hresult.info/FACILITY_NT/0xC000007B).

### Expected behavior:

NVDA should run.

### NVDA logs, crash dumps and other attachments:
N/A

### Further information

Discovered when testing #18814. in https://github.com/nvaccess/nvda/pull/18814#issuecomment-3227651255, @hwf1324 noticed that `vcruntime140.dll` was causing the problem. I inspected `vccorlib140.dll` and `vcruntime140.dll` and determined that they're compiled for x86 machines, which explains why NVDA can't load them. 

#### NVDA installed/portable/running from source:
(Not) running from launcher

#### NVDA version:
Tested with try-fixTouchHandler-52257,05947e24

#### Windows version:
Windows Sandbox:
* Guest: Windows 11 Enterprise Insider Preview Version 24H2 (OS Build 26100.5710)
* Host: Windows 11 Business Insider Preview Version 25H2 (OS Build 26200.5710)

#### Name and version of other software in use when reproducing the issue:
None

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
N/A

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
N/A
